src/components/uicomposition/TilesContainer.astro

```diff
@@ -38,9 +38,9 @@ const {
   }
   <div class="flex w-full flex-col items-start justify-start">
     {
-      headline && headline.length > 0 && (
-        <h2 class="font-serif text-2xl">{headline}</h2>
-      )
+      headline &&
+        (!cta || (cta && cta?.text?.length === 0)) &&
```

Comment:
If text is null it will still not show the headline so it should be this:
```
      headline && (!cta || (cta && (!cta.text || cta.text.length === 0))) && (
```

before:
![image](https://github.com/user-attachments/assets/7cd04a0d-2448-46c5-978a-bbc4d183ac6f)


After:
![image](https://github.com/user-attachments/assets/25574723-3773-4817-a840-4070e56d20fc)

Reply:
fixed it on the main branch